index.d.ts

```diff
@@ -1074,8 +1074,8 @@ declare namespace Eris {
     getPruneCount(guildID: string, options?: GetPruneOptions): Promise<number>;
     pruneMembers(guildID: string, options?: PruneMemberOptions): Promise<number>;
     getVoiceRegions(guildID: string): Promise<VoiceRegion[]>;
-    getInvite(inviteID: string, withCounts?: boolean): Promise<RESTInvite>;
-    acceptInvite(inviteID: string): Promise<RESTInvite>;
+    getInvite(inviteID: string, withCounts?: boolean): Promise<PrivateInvite | GuildInviteWithoutMetadata>;
+    acceptInvite(inviteID: string): Promise<PrivateInvite | GuildInviteWithoutMetadata>;
```

Comment:
as suggested by @Apacheli I'll make a new alias, `AnyInviteWithoutMetadata`, to use in these positions so we don't have unions in the method definitions